packages/@wdio_electron-utils/src/index.ts

```diff
@@ -5,6 +5,7 @@ import { pathToFileURL } from 'node:url';
 import { allOfficialArchsForPlatformAndVersion } from '@electron/packager';
 import findVersions from 'find-versions';
 import type { NormalizedReadResult } from 'read-package-up';
+import { findAndReadConfig, type ReadConfigRequest } from 'app-builder-lib/out/util/config/load.js';
```

Comment:
Perhaps [`getConfig`](https://github.com/electron-userland/electron-builder/blob/3d8e402f0ff1930de9c009dd78ee6dd8fe9c6133/packages/app-builder-lib/src/util/config/load.ts#L90-L96) is more appropriate since it is referenced in multiple places in electron-builder.
<img width="333" alt="image" src="https://github.com/user-attachments/assets/805a7d2e-e506-4079-b324-053a9f07f7da">

Reply:
After looking at the approach here I think it makes more sense to remove the dependency on `app-builder-lib` - Rolling our own isn't a lot of code and I think there's a high likelihood of `electron-builder` breaking things with a future release.